app/scripts/keyring-controller.js

```diff
@@ -188,7 +188,7 @@ class KeyringController extends EventEmitter {
       switch (type) {
         case 'Simple Key Pair':
           let isNotIncluded = !accounts.find((key) => key === newAccount[0] || key === ethUtil.stripHexPrefix(newAccount[0]))
-          return (isNotIncluded) ? Promise.resolve(newAccount) : Promise.reject(new Error('The account your are trying to import is a duplicate'))
+          return (isNotIncluded) ? Promise.resolve(newAccount) : Promise.reject(new Error('The account you\'re are trying to import is a duplicate'))
```

Comment:
When I'm writing a string that needs an apostrophe, I like to use double quotes or back-ticks, but that's just me.